### PR TITLE
fix(SRVKP-4592 & SRVKP-5493): Bug fixes and enhancements for graph reporting

### PR DESCRIPTION
**Pull request by rh-rahulshetty**
This PR adds the following changes:

1. Capturing two new measurements for etcd from Prometheus - `etcd_mvcc_db_total_size_in_use_in_bytes_average` and `etcd_server_quota_backend_bytes_average` (More details on the measurement: [here](https://docs.openshift.com/container-platform/4.13/scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.html#etcd-defrag_recommended-etcd-practices)). The new measurement will tell us the "actual" usage (after compaction) for etcd db size.
2.  Add two new counts `prs_failed` and `trs_failed` in `benchmark-stats.csv`. The measurement captured in `results.PipelineRuns.count.failed` and `results.TaskRuns.count.failed` do not provide the complete count as they count the PRs/TRs post the test run. Any information about failed PR/TR deleted through pruner would be lost. Hence this metric can help find the actual failed count for scenarios with pruner.
3. Added a new script `tools/convert-benchmark-stats.py` that consolidates the run stats from namespace-level to cluster-level. This is used in`tools/visualize-benchmark-stats.sh` to generate the stat plots.

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/lgtm

File: tools/visualize-benchmark-stats.sh [hunk 1]
```diff
@@ -23,13 +28,14 @@ data_file="${1}"
...
-    echo -n " '$data_file' using 2:15 title 'prs_terminated' linewidth 2,"
+    echo -n " '$data_file' using 2:5 title 'prs_failed' linewidth 2,"
+    echo -n " '$data_file' using 2:6 title 'prs_pending' linewidth 2,"
+    echo -n " '$data_file' using 2:7 title 'prs_running' linewidth 2,"
+    echo -n " '$data_file' using 2:8 title 'prs_finished' linewidth 2,"
+    echo -n " '$data_file' using 2:9 title 'prs_signed_true' linewidth 2,"
+    echo -n " '$data_file' using 2:11 title 'prs_finalizers_present' linewidth 2,"
+    echo -n " '$data_file' using 2:15 title 'prs_deleted' linewidth 2,"
+    echo -n " '$data_file' using 2:16 title 'prs_terminated' linewidth 2,"
 } | gnuplot
```
Comment (jhutar) [inline tools/visualize-benchmark-stats.sh:39]: At some point, we maybe want to use your yaml-configured tool to lot these graphs

**Review by jhutar — APPROVED** [review]
Nice!

File: tools/visualize-benchmark-stats.sh (hunk 1 above)
Comment (rh-rahulshetty) [reply tools/visualize-benchmark-stats.sh:39]: Yep, that would be the long term plan. I'll start working on that utility at the side.

**Pull request merged**